saas/wizard/create_build_by_template_view.xml

```diff
@@ -9,8 +9,8 @@
             <form>
                 <group>
                     <field name="build_name"/>
-                    <field name="template_operator_id"/>
-                    <field name="choose_by_random" widget="toggle_button" options='{"active": "Random", "inactive": "Not Random"}'/>
+                    <field name="template_operator_id" domain="[('template_id', '=', context.get('template_id')), ('state', '=', 'done')]"/>
```

Comment:
Shall it be hidden when "random" is selected?

Reply:
in case the saas.template.operator records for this template instance are not created the user will not quite understand why he cannot continue

Reply:
ok. What about widget ``radio`` + readonly mode when ``random`` is selected?

Reply:
Though it doesn't allow to create new record :thinking:

Reply:
Well, we can new fields "template_operator_count" and show ``random`` only when `template_operator_count >1 `